### PR TITLE
[FIX] Inventory: Fixing reordering rules navigation

### PR DESCRIPTION
**Pull request by benderliz**
Doc [task # 4411255](https://www.odoo.com/odoo/project/3835/tasks/4411255)

### CODE REVIEW

**Comment by benderliz** [comment]
Hi @jero-odoo  - I just realized, since this is 17.0, I need to leave some of the screenshots / mentions of consumables/storables. Going to quickly upload a new commit + squash to this PR, so please stay tuned for now. I'll leave a new message when this is ready for your review :)

**Comment by benderliz** [comment]
> Hi @jero-odoo - I just realized, since this is 17.0, I need to leave some of the screenshots / mentions of consumables/storables. Going to quickly upload a new commit + squash to this PR, so please stay tuned for now. I'll leave a new message when this is ready for your review :)

Okay @jero-odoo, this PR is now officially ready for review! I just fixed the section titled, "Create new reordering rules" to correct the navigation instructions for new reordering rules, and also some of the field references. 

Since this was just to make a minor fix for 17.0, I will need to revise this whole doc with new screenshots for 18.0 (but that will be a separate PR).

Thank you so much, and please lmk if there are any questions!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst:71]: I like that you added more context to the UI here. I think for this particular operation, the **On Hand** and **Forecast** values are not editable for a **New** rule line (on my screen it's grayed out and the cursor goes right to the **Route** field, screenshot below)...

Screenshot 2024-12-18 at 4 54 46 PM

...which, makes sense since **Route** **Min Quantity** **Max Quantity** and **To Order** are the inputs directly tied to the RR, whereas the other two are more closely tied to making an inventory adjustment.

My suggestion is to remove these list items, or optionally add a **see also** admonition that points to any relevant docs on how to make an inventory adjustment.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst:73]: Suggestion to rename this to something inventory or warehouse-specific, since in some business cases manufacturing is not involved in their ops, such as with product resellers.

```suggestion
  :doc:`supply-chain route <../../shipping_receiving/daily_operations/use_routes>` for this product
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst:83]: Good suggestion.
Can throw that in a **Tip** admonition if you like to add a splash of color to the page 😉

**Review by StraubCreative — APPROVED** [review]
Hi @benderliz 
Thank you for addressing the task feedback so quickly!
Approving with a few comments below; please address what you agree with and can merge in when you're done 👍 

Also, for this:
> I will need to revise this whole doc with new screenshots for 18.0

Feel free to make the similar, targeted navigational revision in 18.0 as a new quick PR like this one (another 1-2 point), and then for that larger revision (or complete rewrite), do as _a follow-up PR to that_ for another 3 or 5 points. 

Although related, these are 3 separate tasks (2 address the community feedback, and 1 is your own initiative) so if we similarly execute them in this modular way it becomes a) easier for us to track changes and b) you get the full point value 🤙 

..
@robodoo delegate=benderliz

**Comment by StraubCreative** [comment]
@fw-bot ignore up to saas-17.4

**Comment by benderliz** [comment]
> Hi @benderliz Thank you for addressing the task feedback so quickly! Approving with a few comments below; please address what you agree with and can merge in when you're done 👍
> 
> Also, for this:
> 
> > I will need to revise this whole doc with new screenshots for 18.0
> 
> Feel free to make the similar, targeted navigational revision in 18.0 as a new quick PR like this one (another 1-2 point), and then for that larger revision (or complete rewrite), do as _a follow-up PR to that_ for another 3 or 5 points.
> 
> Although related, these are 3 separate tasks (2 address the community feedback, and 1 is your own initiative) so if we similarly execute them in this modular way it becomes a) easier for us to track changes and b) you get the full point value 🤙
> 
> .. @robodoo delegate=benderliz

Hi @StraubCreative! Thanks so much for the quick review, and I appreciate all the direction. That makes a lot of sense - after making revisions/merging this one, I will open a PR for the navigational change in 18.0, and then I'll open a new task for making longer fixes in a separate PR. Cheers!

**Comment by benderliz** [comment]
@robodoo r+

**Pull request closed without merging**
